### PR TITLE
dev-util/roctracer: performance tracing for AMD GPUs

### PR DESCRIPTION
**Pull request by heroxbd**
@candrews @justxi Please check the included 2 ebuilds.  dev-python/CppHeaderParser is a dependency of roctracer.

### CODE REVIEW

File: dev-python/CppHeaderParser/CppHeaderParser-2.7.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Parse C++ header files and generate a data structure"
+HOMEPAGE="http://senexcanis.com/open-source/cppheaderparser/"
```
Comment (vaukai) [inline dev-python/CppHeaderParser/CppHeaderParser-2.7.4.ebuild:10]: HOMEPAGE could use https

**Comment by justxi** [comment]
@heroxbd 
Again nothing to comment. Nice improvements =).

File: dev-python/CppHeaderParser/CppHeaderParser-2.7.4.ebuild (hunk 1 above)
Comment (heroxbd) [reply dev-python/CppHeaderParser/CppHeaderParser-2.7.4.ebuild]: Thanks, updated.

**Comment by heroxbd** [comment]
> @heroxbd
> Again nothing to comment. Nice improvements =).

Thanks!

**Pull request closed without merging**